environs/bootstrap/config.go

```diff
@@ -241,7 +246,7 @@ var configChecker = schema.FieldMap(schema.Fields{
 	CACertKey + "-path":        schema.Omit,
 	CAPrivateKeyKey:            schema.Omit,
 	CAPrivateKeyKey + "-path":  schema.Omit,
-	ControllerServiceTypeKey:   schema.Omit,
+	ControllerServiceTypeKey:   DefaultControllerServiceType,
```

Comment:
I think it better to keep this as schema.Omit so we don't pollute IAAS config.
We can add the default later during the bootstrap config processing only if k8s controller.

Reply:
ah, right